js/todo.js

```diff
@@ -0,0 +1,24 @@
+$(function () {
+    // Mark complete button
```

Comment:
I'm not convinced this comment adds more than the 'button.mark-complete' selector tells us.
If you do think it's unclear, the way to do this with self documenting code is to define functions with meaningful names